### PR TITLE
initiliazed Firestore DB and set up key

### PR DESCRIPTION
**Pull request by joshisacow**
created github secret

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
 from flask import Flask, jsonify, request, abort, make_response
 from flask_restful import Resource, Api, reqparse
+from firebase_admin import credentials, firestore, initialize_app
 
 app = Flask(__name__)
 api = Api(app)
 
+# Initialize Firestore DB
```
Comment (haiyangc) [inline app.py:8]: Start organizing your code. Move all database actions into a separate file. Something like /datasource/firebase.py

File: app.py [hunk 2]
```diff
@@ -18,6 +18,41 @@
...
+            doc = db.collection('users').document(username).get()
+            if not doc.exists:
+                return "user does not exist", 404
+            if doc.to_dict()['password'] != password:
+                return "incorrect password", 401
+            return "login successful", 200
+        except:
+            return "failed to login"
+    
+    return make_response(render_template('login.html')), 200
```
Comment (haiyangc) [inline app.py]: Lol api calls don't return html. Always a json

Comment (haiyangc) [reply app.py]: You can't have ui with api calls. The ui is a separate app. The api will only return success or error and it's up to the ui service to render the login screen and stuff

**Review by haiyangc — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (joshisacow) [reply app.py]: im gonna make a separate pull request for the auth stuff, but I moved the database stuff into a separate file. Where do i call the database initialization steps? Is that in app.py? I currently have anything related to databases in my new Trips.py file, but realized that I probably need it for the auth portion so I probably need to move it to app.py?

Comment (joshisacow) [reply app.py]: nevermind i put it all in app.py and it worked

File: resources/Trips.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from firebase_admin import credentials, firestore, initialize_app, get_app
+from flask import Flask, jsonify, request, abort, make_response, render_template
+from flask_restful import Resource, reqparse
+
+app = Flask(__name__)
+
+# Initialize Firestore DB
+db = firestore.client()
```
Comment (haiyangc) [inline resources/Trips.py:8]: Move your firestore stuff to a different file. That way other endpoints can reuse the same functions to store and retrieve data

File: templates/login.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (haiyangc) [inline templates/login.html:1]: Get rid of all this html stuff

**Pull request closed without merging**